package.json

```diff
@@ -73,7 +73,7 @@
   },
   "devDependencies": {
     "@rollup/plugin-commonjs": "^22.0.2",
-    "@rollup/plugin-node-resolve": "^14.0.0",
+    "@rollup/plugin-node-resolve": "^15.0.1",
     "aws-sdk": "^2.1210.0",
     "chokidar": "^3.5.3",
```

Comment:
Do you want to include the standard codeship scripts?

https://github.com/PropellerAero/cesium/blob/main/package.json#L126-L127